config/m4/cuda.m4

```diff
@@ -1 +1,86 @@
-AM_CONDITIONAL([HAVE_CUDA], [true])
+#
+# Copyright (C) Mellanox Technologies Ltd. 2001-2017.  ALL RIGHTS RESERVED.
+# See file LICENSE for terms.
+#
+
+#
+# Check for CUDA support
+#
+cuda_happy="no"
+gdrcopy_happy="no"
+
+AC_ARG_WITH([cuda],
+           [AS_HELP_STRING([--with-cuda=(DIR)], [Enable the use of CUDA (default is no).])],
```

Comment:
why default is "no"? i think default should be "yes"/"autodetect", same as with other TLs

Reply:
done